### ISSUE TITLE
Erweiterung Interface

### ISSUE THREAD

**Issue by Macsteck** [jandahar/Cards#3]
an das interface zu den Control-Elementen sollte auch jeweils dass ganze CardSet übergeben werden. Dann kann sich das ControlElement bei Bedarf beliebig viele andere Elemente dazu holen können.
@jandhar:
Was meinst du?

**Comment by jandahar** [comment]
Zwischen GUI und logic sollte eine auf Text basierte Schnittstelle bestehen bleiben. Das könnte man dann sehr einfach in HTML requests umwandeln und einen Webservice schreiben, der z.B. ein Java Frontend ermöglicht.

Daher sollte ClCard in der Logik bleiben und die einzige öffentliche Schnittstelle über ClLogic laufen. Dann bleibt es übersichtlicher.

Außerdem wird das Testen einfacher. Unittests stellen eine XML und können mit Funktionsaufrufen mit Klartextstrings arbeiten.

Ist aber trotzdem nochmal eine Diskussion wert.

**Comment by jandahar** [comment]
Ja, wobei es mir wichtig ist, dass das textbasiert bleibt um flexibler zu bleiben, was andere GUI Technologien (z.B. Java Webservice) angeht.
